etc/environment.yml

```diff
@@ -4,7 +4,7 @@ channels:
 dependencies:
   # required
   - python>=3.8
-  - numpy
+  - numpy<2.0
```

Comment:
I think, we are supporting numpy 2 now... or have you detected issues?

Reply:
No issues - this was a legacy thing that snuck in.  We are good with np > 2 yea?

Reply:
I "_think_" so